templates/default/ca-timedrift-metric-script.erb

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+namespace="DECustom"
+region="us-east-1"
+metric_name="CA-timedrift"
+dimension_name="CAName"
+
+ca_list=$(curl -v --user "<%= @ca_stats_user %>:<%= @ca_stats_passwd %>" "<%= @ca_stats_json_url %>" | grep address | awk  -F'"' '{print $4;}' | awk -F. '{print $1;}')
+
+for ca in ${ca_list}
```

Comment:
how many capture agents are we looping through here? I'm concerned about API throttling with this many rapid calls. there are methods by which you can bundle the metric data into a json file and publish with one call, but it would introduce some complexity. Was this tested with the full list of CAs?